skrub/tests/test_joiner.py

```diff
@@ -30,7 +20,7 @@ def test_joiner(px):
         }
     )
 
-    aux_table = px.DataFrame(
+    aux_table = df_module.make_dataframe(
```

Comment:
in theory we should strive to stick to generic functions only so below `ns.shape(big_table)`, `ns.to_numpy(ns.col(big_table, "Population"))` instead of the direct calls to the libraries (which only happen to work because they use the same names for this).

Reply:
You're right, I've changed them
I also need a `to_list` for but you're already implementing it in https://github.com/skrub-data/skrub/pull/895